packages/release-please/src/release-please.ts

```diff
@@ -368,6 +368,15 @@ async function runBranchConfiguration(
   await Runner.createPullRequests(manifest);
 }
 
+interface GitHubCommit {
+  message: string;
+}
+// Helper function to determine if list of commits includes something that
+// looks like a release.
+function hasReleaseCommit(commits: GitHubCommit[]): boolean {
+  return !!commits.find(commit => commit.message.includes('release'));
```

Comment:
I'm worried about false positives here.  Is there a more precise way?

Reply:
Added an author check